ssg/ansible.py

```diff
@@ -50,3 +56,137 @@ def remove_trailing_whitespace(ansible_src):
     Removes trailing whitespace in an Ansible script
     """
     return re.sub(r'[ \t]+$', '', ansible_src, 0, flags=re.M)
+
+
+def _strings_to_list(one_or_more_strings):
+    """
+    Output a list, that either contains one string, or a list of strings.
+    In Python, strings can be cast to lists without error, but with unexpected result.
+    """
+    if isinstance(one_or_more_strings, str):
+        return [one_or_more_strings]
+    else:
+        return list(one_or_more_strings)
+
+
+def update_yaml_list_or_string(current_contents, new_contents):
+    result = []
+    if current_contents:
+        result += _strings_to_list(current_contents)
+    if new_contents:
+        result += _strings_to_list(new_contents)
+    if not result:
+        result = ""
+    if len(result) == 1:
+        result = result[0]
+    return result
+
+
+class AnsibleRemediation(object):
+
```

Comment:
Do you think that this script cans also replace the `@RULE_TITLE@` macro?

Reply:
The proper solution to this is replacing those with jinja macros. As it is easy, but not trivial, I propose to consider it out of scope of this PR.